### PR TITLE
Local Benchmarking

### PR DESCRIPTION
**Pull request by aryszka**

### CODE REVIEW

File: skptesting/benchmark-compress.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	log benchmark-compress.sh [duration] [connections] [warmup-duration]
+	exit 0
+fi
+
+source $GOPATH/src/github.com/zalando/skipper/skptesting/benchmark.inc
+
+trap cleanup SIGINT
+
+log [generating content]
+lorem
```
Comment (lmineiro) [inline skptesting/benchmark-compress.sh]: external dependency? couldn't see a reference in the readme

File: skptesting/benchmark-compress.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+if [ "$1" == -help ]; then
+	log benchmark-compress.sh [duration] [connections] [warmup-duration]
+	exit 0
+fi
+
+source $GOPATH/src/github.com/zalando/skipper/skptesting/benchmark.inc
+
+trap cleanup SIGINT
+
+log [generating content]
```
Comment (lmineiro) [inline skptesting/benchmark-compress.sh]: is this a bash builtin? `log`?

File: skptesting/benchmark.inc [hunk 3]
```diff
@@ -49,52 +49,52 @@ cleanup() {
...
+	tlskey=
+	tlscert=
+	if [ "$3" = tls ]; then
+		tlskey=key.pem
+		tlscert=cert.pem
+	fi
+
+	skipper -access-log-disabled -address "$1" -routes-file "$cwd"/"$2" -insecure \
+		-tls-key "$tlskey" -tls-cert "$tlscert" 2> \
 		>(grep -v 'write: broken pipe' | \
```
Comment (lmineiro) [inline skptesting/benchmark.inc]: grep -E -v "pattern1|pattern2|..."

File: skptesting/readme.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
+# Extended Testing/Benchmarking
+
+This directory may contain additional testing tools and data that fall outside of the standard Go testing
+methods. The tools in this directory may have dependencies that come with limitations on the platform and
+environment that they are running in.
+
+Some of the scripts contain commented instructions to be able to compare the results with other similar
+utilities, e.g. nginx. Feel free to mod these scripts.
+
+WARNING: in the following benchmarks, some of the error messages are muted, which means that when done, the
```
Comment (lmineiro) [inline skptesting/readme.md]: "... which means that, when done, ..." - comma before when done

File: skptesting/readme.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+skptesting/benchmark-static.sh 12 128 3
+```
+
+Benchmarks skipper as a static file server, running for 12s, over 128 connections and with a preliminary warmup
+time of 3s.
+
+## Benchmark Compress
+
+```
+skptesting/benchmark-static.sh 12 128 3
```
Comment (lmineiro) [inline skptesting/readme.md]: s/static/compress (copy&paste)

File: skptesting/readme.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+## Print Memory Profile
+
+```
+skptestint/print-mem-profile.sh
+```
+
+Prints the CPU profile saved from the last run of skptesting/profile-proxy.sh.
+
+## Self Certificate
```
Comment (lmineiro) [inline skptesting/readme.md]: Self-signed certificate

File: skptesting/benchmark.inc (hunk 3 above)
Comment (aryszka) [inline skptesting/benchmark.inc]: ok, thanks

**Comment by lmineiro** [comment]
lgtm

**Pull request merged**